### PR TITLE
Changing to CommonJS -- simple cases

### PR DESCRIPTION
**Pull request by dmitriz**
As suggested in #643 
Single export cases are treated.

### CODE REVIEW

**Comment by bitinn** [comment]
This is a good direction I might consider, but can't merge at the moment due to various implications.

Also changes will have to be made on v3 branch. Master branch is v2 only for now.

**Comment by dmitriz** [comment]
> This is a good direction I might consider, but can't merge at the moment due to various implications.

Sounds mysterious. I've thought the project was looking for new contributions... did I misunderstand?

> Also changes will have to be made on v3 branch. Master branch is v2 only for now.

Any reasoning behind? The change is not breaking, not even a change in functionality.

**Comment by bitinn** [comment]
> Sounds mysterious.

TL;DR: I can't merge it as the actual export isn't a part of the tests. I can't verify this to work in the wild. (which users can import using named or commonjs).

And if the goal is to remove `babel`, then there is more work to check it actually works and provide a migration plan for cases that doesn't.

I do agree 2.x was doing too much and hence making our work more difficult.

**Comment by bitinn** [comment]
Here is my suggestion to move forward:

- Make the same PR against `v3` branch.
- Try to test different ways to import `node-fetch` if you can.
- After we merged this and #640, we can publish v3 alpha and try to iterate further on this.

(I would love to only deal with commonjs export, but v2 is not the place to do it.)

**Comment by Richienb** [comment]
In the new [roadmap](https://github.com/bitinn/node-fetch/issues/668), we are using replacing Babel and Rollup with Microbundle.
